### ISSUE TITLE
I cannot increase the maximum number of messages to 10

### ISSUE THREAD

**Issue by ExpErgio** [Androz2091/discord-backup#237]
Good night.
I am trying to keep it in the backup, maximum 5,000 messages per channel; but having edited it, it still keeps only 10.

I do not know if I can be wrong when declaring it. I hope you can help me

.
.
.
await backup.create(message.guild, {
        jsonBeautify: true,
        maxMessagesPerChannel: 5000
    }).then(backupData => {
.
.
.

**Comment by Positive2022** [comment]
did u forgot to restart the bot?

**Comment by brentleywilson** [comment]
Im having the same issue, how do u increase the maximum backup ? when i load a backup it only fills 10 messages per channel

**Comment by sw3do** [comment]
I don’t think this issue should occur with the latest version of the discord-backup module. However, if you still experience this problem, please don’t hesitate to let us know. We’ll be happy to help!
